Civi/Share/Message.php

```diff
@@ -26,12 +28,13 @@ class Message {
   /** @var array list of change ids */
   protected array $change_ids = [];
 
+  protected Client $civiMRFClient;
+
   /**
    * Create a new, empty message
    */
-  public function __construct()
-  {
-
+  public function __construct() {
+    $this->civiMRFClient = Civi::service('civi.share.civimrf');
```

Comment:
Inject `$civiMRFClient` in constructor. Naming it `cmrfClient` would be consistent with `cmrfCore` in `Client`.